lib/tasks/zomato.rake

```diff
@@ -0,0 +1,21 @@
+require_relative '../zomato'
+
+namespace :zomato do
+  desc 'Load cuisines'
+  task :cuisines, [:city_id]  => :environment do |task, args|
+    Zomato.new(args[:city_id].to_i).createCuisines
+  end
+
+  desc 'Load Restaurants'
+  task :restaurants, [:city_id, :from, :to, :count, :review_count] => :environment do |task, args|
```

Comment:
Just curious, why you need all these variables in the rake task? Especially `to` and `review_count`?

Reply:
just to have more control over what gets loaded, could be hard coded / ENV, not critical